Pod/Core/StringGherkinExtension.swift

```diff
@@ -17,12 +17,12 @@ public extension String {
 
     */
     var camelCaseify: String {
-        get {
-            guard case let characters = (self.split { $0 == " " || $0 == "-" }), characters.count > 1 else {
-                return self.uppercaseFirstLetterString
-            }
-            return characters.map { String($0).lowercased().uppercaseFirstLetterString }.joined(separator: "")
+        let allowed = CharacterSet(charactersIn: "-").union(.alphanumerics).union(.whitespaces)
```

Comment:
Are we considering how this might affect current users of this in other languages (e.g. Français) - how does it deal with accents on alphanumeric characters?

Reply:
I'd expect `alphanumeric` characters to include those too, but worth writing a unit test for that 👍

Reply:
```swift

let set = CharacterSet(charactersIn: "-").union(.alphanumerics).union(.whitespaces)

set.contains("Ḟ") // true
set.contains("ê") // true
```

LGTM